tutorials/integrations/zapper.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Portfolio Tracking on Moonbeam with Zapper
+description: Learn how to track your holdings on Moonbeam using Zapper and interact with different DeFi platforms on the Moonbeam Ecosystem.
```

Comment:
So you can interact with the DeFi platforms through Zapper? Like sending transactions? Or is it just viewing activity on the various platforms?

Reply:
This should say: "in the Moonbeam ecosystem."

Use "in" instead of "on" and no need to capitalize "ecosystem"

Reply:
Ok, sounds good. Thanks!